metadata.json

```diff
@@ -10,7 +10,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.16.0 < 9.0.0"
+      "version_requirement": ">= 4.16.0 < 10.0.0"
```

Comment:
Hum, `stdlib::ensure_packages` appeared in `9.0.0` (where `merge` is also available as `stdlib::merge`).  Not sure if we should bump the minimum version to 9.0.0 or rather use the deprecated functions.

Reply:
I prefer to do it now and rip out all the legacy function calls. Otherwise we need to touch the codebase in the future as well.

Reply:
ACK! I opened #1114 to update the minimum version requirement.